rmgpy/data/thermo.py

```diff
@@ -193,16 +193,19 @@ def addThermoData(thermoData1, thermoData2, groupAdditivity=False):
         if len(thermoData1.Tdata.value_si) != len(thermoData2.Tdata.value_si) or any([T1 != T2 for T1, T2 in zip(thermoData1.Tdata.value_si, thermoData2.Tdata.value_si)]):
             raise Exception('Cannot add these ThermoData objects due to their having different temperature points.')
         
+        testZero = abs(thermoData2.H298.value_si)+abs(thermoData2.S298.value_si) # Used to check if all of the entries in thermoData1 are zero
```

Comment:
A more concise implementation would be:

testZero=sum([abs(value) for value in [H298, S298] + Cpdata]]

Then just check if this is zero.